src/status_im2/contexts/onboarding/profiles/view.cljs

```diff
@@ -178,7 +179,11 @@
   (let [{:keys [key-uid name customization-color error
                 processing password]} (rf/sub [:multiaccounts/login])
         sign-in-enabled?              (rf/sub [:sign-in-enabled?])
-        profile-picture               (rf/sub [:multiaccounts/login-profiles-picture key-uid])]
+        profile-picture               (rf/sub [:multiaccounts/login-profiles-picture key-uid])
+        error                         (if (or (= error "file is not a database")
+                                              (string/starts-with? error "failed to set "))
+                                        (i18n/label :t/wrong-password)
+                                        error)]
```

Comment:
Not an ideal solution IMO, we should have a better error system on status-go, but implemented same approach as status-desktop as of now (https://github.com/status-im/status-desktop/pull/10840/files)

Reply:
https://github.com/status-im/status-go/issues/3576